### PR TITLE
Always use GetEncoding(0) to try getting the default encoding

### PR DESCRIPTION
**Pull request by agocke**

### CODE REVIEW

**Comment by agocke** [comment]
@jaredpar @tmat FYI

File: src/Compilers/Core/Portable/EncodedStringText.cs [hunk 1]
```diff
@@ -32,20 +32,14 @@ private static Encoding GetFallbackEncoding()
...
-                }
-                else
-                {
-                    // If we're running on the desktop framework we should be able
-                    // to get the default ANSI code page in the operating system's
-                    // regional and language settings,
-                    return PortableShim.Encoding.GetEncoding(0)
-                        ?? PortableShim.Encoding.GetEncoding(1252);
                 }
+                // Try to get the default ANSI code page in the operating system's
```
Comment (jaredpar) [inline src/Compilers/Core/Portable/EncodedStringText.cs]: Add a space above.

**Comment by jaredpar** [comment]
:+1:

**Comment by agocke** [comment]
test eta please

**Pull request merged**